specificatie/verblijfplaats.yaml

```diff
@@ -277,8 +251,6 @@ components:
               $ref: '#/components/schemas/Adresregel2'
```

Comment:
de adresregels moeten uit Adres en Locatie worden verwijderd. Die zitten nu in Adressering

Reply:
Aangepast